modin/pandas/test/test_io.py

```diff
@@ -1643,6 +1646,26 @@ def test_read_parquet_2462(self):
 
             df_equals(test_df, read_df)
 
+    @pytest.mark.xfail(
+        condition="config.getoption('--simulate-cloud').lower() != 'off'",
+        reason="The reason of tests fail in `cloud` mode is unknown for now - issue #3264",
+    )
+    def test_read_parquet_glob_path(self):
```

Comment:
We don't need this anymore, right? Shouldn't testing for reading `Path` objects in the previous test case be enough?

Reply:
Hmmm I kinda wanted to keep it there because it tested some of the `glob` stuff as a use case at least. It was good for sanity and is a fairly fast test.

Reply:
I want to be really aggressive about cutting out redundant tests. They may be fast, but they make the files longer so it's harder to find things you're looking for. Also more tests break if you break just one feature. We should pare tests down so they're just enough to cover all the ways you can use a modin API.

Reply:
Ok, fair enough, I'll remove this test.